pkg/admin/prerun/version.go

```diff
@@ -27,6 +28,10 @@ type versionFile struct {
 	BootID       string          `json:"boot_id"`
 }
 
+func (hi *versionFile) BackupName() data.BackupName {
+	return data.BackupName(fmt.Sprintf("%s_%s", hi.DeploymentID, hi.BootID))
+}
```

Comment:
There'a an alternative for getting IDs for previous boot. boot ID from journalctl. Deployment ID is a bit controversial though:
```
[redhat@repro-host1 ~]$ sudo journalctl  -u ostree-prepare-root
Sep 22 05:18:50 localhost systemd[1]: Starting OSTree Prepare OS/...
Sep 22 05:18:50 localhost ostree-prepare-root[600]: preparing sysroot at /sysroot
Sep 22 05:18:50 localhost ostree-prepare-root[600]: Resolved OSTree target to: /sysroot/ostree/deploy/rhel/deploy/7c5bfecf8acf6e547d39efc9ae15f4e1b32df134cb4d2727ffae1c786784cc4f.0
Sep 22 05:18:50 localhost ostree-prepare-root[600]: filesystem at /sysroot currently writable: 0
Sep 22 05:18:50 localhost ostree-prepare-root[600]: sysroot.readonly configuration value: 0
Sep 22 05:18:50 localhost systemd[1]: Finished OSTree Prepare OS/.
```